src/redux/reducer/reservation.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-param-reassign */
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = {
+  data: [],
+  status: 'idle',
+};
+
+export const getReservation = createAsyncThunk(
+  'reservation/getReservation',
+  async () => {
+    const response = await fetch(
+      'https://blue-dreams-back-end.herokuapp.com/reservation',
+      {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          Authorization: localStorage.getItem('token'),
+        },
+      },
+    );
+    if (response.ok) {
+      return response.json();
+    }
+    throw response.json();
+  },
+);
+
+export const postReservation = createAsyncThunk(
+  'reservations/postReservation',
+  async (object) => {
+    const response = await fetch(
+      'https://blue-dreams-back-end.herokuapp.com/reservations/',
```

Comment:
In this the second part it should be https://blue-dreams-back-end.herokuapp.com/reservations/create